packages/perseus/src/widgets/numeric-input/score-numeric-input.test.ts

```diff
@@ -6,13 +6,39 @@ import scoreNumericInput, {maybeParsePercentInput} from "./score-numeric-input";
 
 import type {PerseusNumericInputRubric} from "../../validation.types";
 
-describe("static function validate", () => {
+describe("scoreNumericInput", () => {
     beforeEach(() => {
         jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
             testDependencies,
         );
     });
 
+    it("is correct when input is empty but answer is 1 and coefficient: true", () => {
```

Comment:
The `expression` widget is based on KhanAnswerTypes. And within that file there's a special case where it interprets a completely empty input (aka `""`) as 1! 

So this is a behavioural test to document this because it's _very_ non-obvious unless you trace through the KhanAswerTypes!